modules/cnv-deleting-custom-resource.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_install/uninstalling-container-native-virtualization.adoc
+
+[id="cnv-deleting-kubevirt-hyperconverged-custom-resource_{context}"]
+= Deleting the KubeVirt HyperConverged custom resource
+
+To uninstall {ProductName}, you must delete the custom resource that you created
+during deployment.
+
+.Prerequisites
+
+* An active *KubeVirt HyperConverged Cluster Operator Deployment* custom resource
+
+.Procedure
+
+. Ensure that you are in the `kubevirt-hyperconverged` project. If not, select
+it from the *Project* list in the {product-title} web console.
+
+. Navigate to the *Catalog* -> *Installed Operators* page.
+
+. Click *KubeVirt HyperConverged Cluster Operator*.
+
+. Click the *KubeVirt HyperConverged Cluster Operator Deployment* tab.
+
+. Click the Options menu in the row containing the *kubevirt-hyperconverged*
+custom resource. In the expanded menu, click *Delete Hyper Converged*.
```

Comment:
Is this the kebab? If so, there should be a {kebab} attribute, and it's called "the Options menu {kebab}."

Reply:
s/Hyper Converged/HyperConverged

Reply:
@alexxa I thought that was weird, too. Thanks for reporting the bug!